joomla/joomla-cms
# [update sites rebuild/delete] Only for non core extensions. Solves #10826

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/10826 and https://github.com/joomla/joomla-cms/issues/10857 (update sites part).
#### Summary of Changes

This PR changes updates sites rebuild process so only non core extensions can be deleted/rebuilt.

Also adds a error message on Rebuild when extension joomla plugin is disabled.
#### Testing Instructions
1. Use joomla latest staging
2. Confirm problems described in https://github.com/joomla/joomla-cms/issues/10826 are solved
3. Do a general test in update sites delete/rebuild process with several 3º party extensions with update servers (of several types: templates, frameworks, modules, plugins, components, etc)
4. Disable Joomla Extension plugin
5. Try to Rebuild and check the error message
6. Enable Joomla Extension plugin and repeat step 5. no error
7. Code review
#### Observations

**Mantainers** this hardcodes `joomla`, `pkg_en-GB` and `com_joomlaupdate` has core extensions with update servers.
